### DISCUSSION TITLE
Custom theme

### DISCUSSION THREAD

**Discussion by noahfraiture** [NvChad/NvChad#2355] [Q&A] [answered]
Hello, it may seem like a stupid question, but I couldn't find the answer. The structure of the project seems to have changed since the last discussion, issues, and pull requests that talk about it. Where are the themes located? I'd like to add themes created from small customizations of existing ones. There's no need to create a new one from scratch; I simply want themes that match my terminal colors. Thank you for reading.

**Comment by siduck** [comment, marked as answer]
@NoahFraiture custom/themes dir . read https://nvchad.com/docs/config/theming#local_themes
